### PR TITLE
[New] Entry: Woven Fabric

### PR DESCRIPTION
**Pull request by d-simon**

### CODE REVIEW

File: data/networks/pullover.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Author: David Simon
+# Date: 17-11-2015
+# References:
+# - https://en.wikipedia.org/wiki/Sweater
+# - Image: Freimut Bahlo - Own work / CC BY-SA 3.0
+
+definition:
+  name: Pullover
```
Comment (256dpi) [inline data/networks/pullover.yml]: Maybe also "wool fabric" rather than pullover (would be more general)?

File: data/networks/pullover.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  nodes: Knots
+  edges: Threads
+  mediums:
+    - Cotton
+    - Synthetic Fibers
+
+attributes:
+  topology: decentralized
```
Comment (256dpi) [inline data/networks/pullover.yml]: Isn't it distributed, as no node has more control than another?

**Comment by 256dpi** [comment]
The image could be better, no? ;)

**Comment by d-simon** [comment]
I think the image is perfect. What did you have in mind?

**Comment by 256dpi** [comment]
Less wooden floors and tables and more woven fabric would be nice...

**Comment by d-simon** [comment]
fixed.

**Comment by 256dpi** [comment]
added to #7

**Pull request closed without merging**